package/fa_adept_client.tcl

```diff
@@ -458,6 +459,12 @@ set caDir [file join [file dirname [info script]] "ca"]
 				}
 			}
 
+			"adjust_upload_rate" {
+				if {[info exists faupCommand]} {
+					{*}$faupCommand row
+				}
+			}
+
```

Comment:
How about matching on a pattern here (like mlat does, below) e.g. faup*
since faupCommand already has to know the individual commands